Makefile

```diff
@@ -6,3 +6,4 @@ build:
 	cp ./package.json ./dist/package.json
 	cp ./LICENSE ./dist/LICENSE
 	cp ./README.rst ./dist/README.rst
+	node build-scss.js
```

Comment:
Will this get you the correct version of node anywhere this makefile is supposed to run?

Reply:
Yeah, the node version is set in the `.nvmrc` file.

Reply:
The github actions hardcodes a node version right now, I think we have not made is support .nvmrc yet. but locally with `nvm use`, yes.